docs/docs.yml

```
@@ -111,6 +111,8 @@ navigation:
                 path: wallets/pages/authentication/login-methods/social-login.mdx
               - page: Custom social providers
                 path: wallets/pages/react/login-methods/social-providers.mdx
+              - page: Bring Your Own Authentication
```

Sorry realize we should be consistent here
```suggestion
              - page: Bring your own authentication
```